src/plugins/skills/src/systems/active_slots/track_old_activity.rs

```diff
@@ -0,0 +1,63 @@
+use crate::components::active_slots::{ActiveSlots, Current, Old};
+use bevy::prelude::*;
+
+impl ActiveSlots<Old> {
+	pub(crate) fn track(mut active_slots: Query<(&mut Self, &ActiveSlots<Current>)>) {
```

Comment:
We should filter for `Changed<ActiveSlots<Current>>` to avoid update even if there is no skill execution going on.